### PR TITLE
Remove basic auth and JWT

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/hosting/environment-variables/environment-variables.md [hunk 1]
```diff
@@ -118,7 +117,7 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
 | `N8N_SMTP_SSL` | Boolean | `true` | Whether to use SSL for SMTP (true) or not (false). |
 | `N8N_UM_EMAIL_TEMPLATES_INVITE` | String | - | Full path to your HTML email template. This overrides the default template for invite emails. |
 | `N8N_UM_EMAIL_TEMPLATES_PWRESET` | String | - | Full path to your HTML email template. This overrides the default template for password reset emails. |
-| `N8N_USER_MANAGEMENT_JWT_SECRET` | String | - | Set a specific JWT secret. By default, n8n generates one on start. |
```
Comment (netroy) [inline docs/hosting/environment-variables/environment-variables.md:121]: this is still relevant, as it's used by the JWTs that we generate. We are only removing the **external** JWT support.

**Review by netroy — CHANGES_REQUESTED** [review]
besides the one comment, LGTM

File: docs/hosting/environment-variables/environment-variables.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/hosting/environment-variables/environment-variables.md:121]: Fixed, thanks

**Review by netroy — APPROVED** [review]
Thanks 🙏🏽

**Pull request merged**